pages/consulting/video-production/[filename].tsx

```diff
@@ -0,0 +1,182 @@
+import { InferGetStaticPropsType } from "next";
+
+import { tinaField, useTina } from "tinacms/dist/react";
+import { TinaMarkdown } from "tinacms/dist/rich-text";
+import { client } from "../../../.tina/__generated__/client";
+
+import { ReactElement } from "react";
+import { BuiltOnAzure } from "../../../components/blocks";
+import { Blocks } from "../../../components/blocks-renderer";
+import { Booking } from "../../../components/blocks/booking";
+import { Breadcrumbs } from "../../../components/blocks/breadcrumbs";
+import { componentRenderer } from "../../../components/blocks/mdxComponentRenderer";
+import { BookingButton } from "../../../components/bookingButton/bookingButton";
+import { Layout } from "../../../components/layout";
+import { Container } from "../../../components/util/container";
+import { Section } from "../../../components/util/section";
+import { removeExtension } from "../../../services/client/utils.service";
+
+import ReactDOMServer from "react-dom/server";
+
+export default function VideoProductionPage(
+  props: InferGetStaticPropsType<typeof getStaticProps>
+) {
+  const { data } = useTina({
+    data: props.data,
+    query: props.query,
+    variables: props.variables,
+  });
+
+  const bookingButtonProps = {
+    buttonText: data.global.bookingButtonText,
+  };
+
+  return (
+    <Layout>
```

Comment:
Need to add the SEO stuff

copy consulting template

https://github.com/SSWConsulting/SSW.Website/blob/656c9ac9c78b7fb250a7c33bf61f85eebe521f68/pages/consulting/%5Bfilename%5D.tsx#L59-L60

```suggestion
  return (
        <SEO seo={props.seo} />
    <Layout>
```